tachyon/zk/plonk/examples/BUILD.bazel

```
@@ -27,8 +37,41 @@ tachyon_cc_library(
     deps = ["//tachyon/zk/plonk/constraint_system:circuit"],
 )
 
+tachyon_cc_library(
+    name = "simple_circuit_test_data",
+    testonly = True,
+    hdrs = ["simple_circuit_test_data.h"],
+    deps = [
+        ":circuit_test_data",
+        "//tachyon/zk/plonk/layout/floor_planner:simple_floor_planner",
+        "//tachyon/zk/plonk/layout/floor_planner/v1:v1_floor_planner",
+    ],
+)
+
 tachyon_cc_library(
     name = "simple_lookup_circuit",
     hdrs = ["simple_lookup_circuit.h"],
     deps = ["//tachyon/zk/plonk/constraint_system:circuit"],
 )
+
+tachyon_cc_unittest(
+    name = "example_circuit_tests",
+    srcs = [
+        "example_circuit_tests.cc",
+    ],
```

It would be nice to be inlined!

```suggestion
    srcs = ["example_circuit_tests.cc"],
```